product_listings_manager/app.py

```diff
@@ -7,7 +7,8 @@
 
 # Set products.py's DB values from our Flask config:
 app.config.from_object('product_listings_manager.config')
-app.config.from_envvar('FLASK_CONFIG')
+app.config.from_envvar('FLASK_CONFIG', silent=True)
+app.config.from_pyfile('/etc/product-listings-manager/config.py', silent=True)
```

Comment:
Shouldn't configuration from `FLASK_CONFIG` be loaded last so options from `/etc` can be overridden (in case it's possible to use env)?